apps/prairielearn/src/lib/question-render.sql

```diff
@@ -96,6 +97,7 @@ FROM
     LIMIT
       1
   ) AS gj ON TRUE
+  LEFT JOIN users u ON (s.auth_user_id = u.user_id)
```

Comment:
`submissions.auth_user_id` is unfortunately nullable, so I think technically we should be `LEFT JOIN`ing here. We should almost certainly add a `NOT NULL` constraint to this column in another PR.